src/Confluent.Kafka/AdminClient.cs

```diff
@@ -363,6 +363,63 @@ private ListConsumerGroupsReport extractListConsumerGroupsResults(IntPtr resultP
             return result;
         }
 
+        private DescribeUserScramCredentialsResult extractDescribeUserScramCredentialsResult(IntPtr eventPtr){
+            var result = new DescribeUserScramCredentialsResult();
+            if (Librdkafka.DescribeUserScramCredentials_result_get_errorcode(eventPtr)!= ErrorCode.NoError)
+            {
+                result.Error = new Error(Librdkafka.DescribeUserScramCredentials_result_get_errorcode(eventPtr));
+            }
```

Comment:
If there is an error why aren't we setting the other fields? Aren't they passed on from Librdkafka?